### PR TITLE
Add onEvent action hooks

### PR DESCRIPTION
**Pull request by sbrichards**
Add ability to pass in action handlers for `onEvent` & other callbacks
https://plaid.com/docs/api/#onevent-callback

### CODE REVIEW

File: addon/components/plaid-link.js [hunk 1]
```diff
@@ -44,7 +47,19 @@ export default Ember.Component.extend({
     this._link.open(this.get('institution'));
   },
 
+  _onLoad() {
+    this.sendAction('onLoadAction');
+  },
+
+  _onEvent(eventName, meta) {
+    this.sendAction('onEventAction', eventName, meta);
```
Comment (sbrichards) [inline addon/components/plaid-link.js]: No need to check if `onEventAction` is null, correct?

**Comment by sbrichards** [comment]
@josemarluedke Travis tests failing:
```
modules/ember-plaid/components/plaid-link.js: line 43, col 18, 'Plaid' is not defined.
```
Didn't change that line at all... is there something I need to rebuild before pushing up?

File: addon/components/plaid-link.js (hunk 1 above)
Comment (josemarluedke) [reply addon/components/plaid-link.js]: Nope.

**Comment by josemarluedke** [comment]
@sbrichards About the test, I don't believe we had them passing before. So, I believe you can ship this.

**Pull request merged**